scripts/spack/collisions.patch

```diff
@@ -0,0 +1,12 @@
+--- ori/domain/test/collision_reference/CMakeLists.txt	2021-06-22 16:54:10.697155000 +0200
++++ new/domain/test/collision_reference/CMakeLists.txt	2021-08-23 17:28:57.881742000 +0200
+@@ -1,5 +1,5 @@
+ 
+-add_executable(collisions_a2a_test collisions_a2a.cpp test_main.cpp)
+-target_include_directories(collisions_a2a_test PRIVATE ../../include)
+-target_link_libraries(collisions_a2a_test gtest_main)
+-add_test(NAME CollisionReferenceTests COMMAND collisions_a2a_test)
++# add_executable(collisions_a2a_test collisions_a2a.cpp test_main.cpp)
++# target_include_directories(collisions_a2a_test PRIVATE ../../include)
++# target_link_libraries(collisions_a2a_test gtest_main)
++# add_test(NAME CollisionReferenceTests COMMAND collisions_a2a_test)
```

Comment:
This test has been removed in the Hilbert PR, making this patch obsolete. Probably worth it to merge Hilbert first, then merge develop into this PR.

Reply:
https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app/pull/93/commits/a3786cae97acba5f77633b1960a425d6184d271b